test/state/test_state.hpp

```diff
@@ -72,31 +72,34 @@ class TestState : public state::StateView, public std::map<address, TestAccount>
     void apply(const state::StateDiff& diff);
 };
 
-class TestBlockHashes : public state::BlockHashes
+class TestBlockHashes : public state::BlockHashes, public std::unordered_map<int64_t, bytes32>
 {
     /// The map block_number => block hash of known blocks.
-    const std::unordered_map<int64_t, bytes32>& known_block_hashes_;
+    // const std::unordered_map<int64_t, bytes32>& known_block_hashes_;
```

Comment:
commented lines here and below should be removed instead?

Reply:
thx.